### ISSUE TITLE
Manual metadata uploading script may upload metadata to BQ multiple times (duplicate rows)

### ISSUE THREAD

**Issue by SHuang-Broad** [broadinstitute/cromwell-task-monitor-bq#1]
This can be tricky:
  * one may accidentally run the script multiple times
  * previous upload job(s) may fail, and reruns may lead to partial duplicate rows

**Comment by dinvlad** [comment]
Yes, it's a concern with BigQuery streaming inserts. However, in this case the duplicate rows may not be a significant drawback, since we only care about aggregate stats in the end. Preventing duplications altogether may be quite a difficult thing to do properly.

That being said, BigQuery streaming library already does a reasonable job at that, because by default it assigns a random `insertId` which is valid for something like 1 minute. In practice, I think this prevents most duplications, but I'd suggest you collect some data and check if you see any duplications still. It wasn't a significant issue in our workflows, from what I recall.
